src/components/CompareResults/beta/RevisionHeader.tsx

```diff
@@ -38,10 +42,36 @@ const styles = {
   }),
 };
 
-function createTitle(header: RevisionsHeader) {
-  return header.test === '' || header.suite === header.test
-    ? header.suite
-    : `${header.suite} ${header.test}`;
+function createTitle(
+  header: RevisionsHeader,
+  docsURL: string,
+  isLinkSupported: boolean,
+) {
+  const isTestUnavailable = header.test === '' || header.suite === header.test;
+  if (isLinkSupported) {
+    return isTestUnavailable ? (
+      <Link
```

Comment:
Could you try adding `target="_blank"` as props so that when clicking the link a new tab is opened?